scripts/issue/480/disable_digg_oracle.py

```diff
@@ -0,0 +1,24 @@
+'''
+To update the DIGG Median Oracle (0x058ec2Bf15011095a25670b618A129c043e2162E)
+providers need to push reports. Removing the providers from the oracle prevents
+the price from being updating and prevents rebasing.
+'''
+
+from brownie import interface
+
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry
+
+
+def main():
+    dev = GreatApeSafe(registry.eth.badger_wallets.dev_multisig)
+    oracle = interface.IMedianOracle(
+        registry.eth.oracles.oracle, owner=dev.account
+    )
+
+    for _ in range(oracle.providersSize()):
+        oracle.removeProvider(oracle.providers(0))
```

Comment:
This should be:
```
for i in range(oracle.providersSize()):
    oracle.removeProvider(oracle.providers(i))
```

Reply:
no, because removing the first entry shortens the length of the array

Reply:
https://etherscan.io/address/0x058ec2Bf15011095a25670b618A129c043e2162E#code#L424

Reply:
Interesting - good catch.  They could have learned something from balancer with linked lists to make it more gas efficient :D

This LGTM then!